cms/tests/test_static_analysis.py

```diff
@@ -10,5 +11,6 @@ class AboveStaticAnalysisCodeTest(TestCase):
     def test_pyflakes(self):
         import cms
         import menus
-        errors, message = pyflakes((cms, menus))
-        self.assertEqual(errors, 0, message)
+        if not DJANGO_3_0:
```

Comment:
need a new pyflakes release, as for tests only, just trivial

Reply:
only trivial changes needed here. I also bumped pyflakes for all django versions